README.markdown

```diff
@@ -204,15 +177,13 @@ class TestDatabase : Database {
 
 When they are needed, use inline comments to explain **why** a particular piece of code does something. Comments must be kept up-to-date or deleted.
 
-Avoid block comments inline with code, as the code should be as self-documenting as possible.
```

Comment:
I think this is still a good statement to keep. It doesn't say to not use it, just to try to avoid it.

Reply:
We also voted to get rid of this after some discussion that inline comments have a place and should be used with judgement that didn't warrant a rule against it.